contracts/LaunchpegFactory.sol

```diff
@@ -327,4 +353,36 @@ contract LaunchpegFactory is
         joeFeeCollector = _joeFeeCollector;
         emit SetDefaultJoeFeeCollector(_joeFeeCollector);
     }
+
+    function addDefaultPauser(address _pauser)
+        external
+        override
+        onlyOwner
+        returns (bool)
+    {
+        return _defaultPausers.add(_pauser);
+    }
+
+    function removeDefaultPauser(address _pauser)
+        external
+        override
+        onlyOwner
+        returns (bool)
+    {
+        return _defaultPausers.remove(_pauser);
+    }
+
+    function _setPausers(address launchpeg) private {
+        bytes32 pauserRole = ISafePausableUpgradeable(launchpeg).PAUSER_ROLE();
+        bytes32 unPauserRole = ISafePausableUpgradeable(launchpeg)
+            .UNPAUSER_ROLE();
+        for (uint256 i; i < _defaultPausers.length(); i++) {
+            address pauser = _defaultPausers.at(i);
+            IAccessControlUpgradeable(launchpeg).grantRole(pauserRole, pauser);
+            IAccessControlUpgradeable(launchpeg).grantRole(
```

Comment:
We only need the `pauser` role for these default pausers. The idea is that we add EOA wallets from a few people on the team to be able to pause a contract quickly, but then it needs to be unpaused by the multisig for safety reasons

Reply:
updated!